README.md

```diff
@@ -59,6 +59,25 @@ client.on('text', (packet) => { // Listen for chat messages and echo them back.
 })
 ```
 
+### Client example joining a Realm
+
+Example to connect to a Realm that the authenticating account is owner of or has been invited to:
+
+```js
+const bedrock = require('bedrock-protocol')
+const client = bedrock.createClient({
+  realms: { // Only one option needs to be present
```

Comment:
Can you only demonstrate one specific way to do this here? The other options should be explained in the API docs. This makes it possible to copy paste from here without changing the code.